pkg/chartutil/requirements.go

```diff
@@ -62,6 +62,8 @@ type Dependency struct {
 	Tags []string `json:"tags"`
 	// Enabled bool determines if chart should be loaded
 	Enabled bool `json:"enabled"`
+	// ImportValues holds the mapping of source values to parent key to be imported
+	ImportValues []interface{} `json:"import-values"`
```

Comment:
This is a list or sublist depending on the format. I updated the comment to explain more.